.github/workflows/nightly-build.yml

```diff
@@ -1,9 +1,10 @@
-name: Nightly build
+name: Build and push to quay.io the latest code
```

Comment:
Provided that it is no longer a nightly but per-push, shall we also rename the file `nightly-build.yml`?

Reply:
Well I will keep it like that because it still push nightly tag :)
I changed it in zync

Reply:
Although I recognize that many projects also call nightly builds, builds that are performed more often that once a day. So this is just a question to consider. It's fine with me to keep as is.